packages/asset-uploader/src/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const DEFAULT_UPLPOAD_PATH = "uploads";
+export const S3_ENV_KEYS = [
```

Comment:
looks like those are not all keys, but only the once that are required, would call it then 
```suggestion
export const S3_REQUIRED_ENV_KEYS = [
```

Reply:
the entire thing with the keys looks like something that will be quite a bit to manage, I would really make a zod schema with optional keys and defaults in one central place, then call it once and export everything as a single config object where we have everything validated and in one place for the s3 uploader